packages/ember-glimmer/lib/utils/references.js

```diff
@@ -52,6 +52,27 @@ export class UpdatableReference extends RootReference {
   }
 }
 
+export class GetHelperReference {
+  constructor(args) {
+    this.positionalArgs = args.positional;
+  }
+
+  isDirty() { return true; }
+
+  value() {
+    let objRef = this.positionalArgs.at(0).value(),
+      pathRef = this.positionalArgs.at(1).value();
+
+    return objRef ? get(objRef, pathRef) : objRef;
```

Comment:
I think it's a requirement of this helper to have 2 arguments passed https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/keywords/get.js#L89-L90. This assertion needs to be brought back. It will also simplify this code and actually becomes pretty simple I believe.

```
export class GetHelperReference {
  constructor(args) {
    this.positionalArgs = args.positional;
  }

  isDirty() { return true; }

  value() {
    let objRef = this.positionalArgs.at(0).value();
    let pathRef = this.positionalArgs.at(1).value();
    return  get(objRef, pathRef);
  },
  get(key) {
   return get(this.value(), key);
  }
}
```